WTWSMS/history/titles/c_aracena.txt

```diff
@@ -13,10 +13,6 @@
 	holder=36553 # Opimius Gaius Comenciolus   
 	liege = e_byzantium
 }
-560.1.1={ 
-	holder=36558 # Flavius Constans    
-	liege = 0
-}
```

Comment:
Why did you remove this entry?

Reply:
If I remember well, he was rebel "duke of Corduba". It's impossible for a small rebel army to have controlled more than Corduba.

Reply:
If the rebels controlled the important city of Cordoba I don't see the issue for them to hold countryside. More importantly though I think the question is what sources justified the change.

Reply:
Check the map we use: data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMSEhMTEhMSFhUXGRkYGBUYGRIYFRoYGBgYFxcXGBYYHigjGh0lGxgXITEhJSkrLi4uFyAzODMtNygtLisBCgoKDg0OGxAQGy0mICYtLS0tKzArLS0tLS0tLS0tLS0tLS0tLS8tLy0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIAOEA4QMBEQACEQEDEQH/xAAbAAEAAgMBAQAAAAAAAAAAAAAABQYCAwQBB//EAD0QAAEDAgMFBwIEBAYCAwAAAAEAAhEDIQQSMQUTQVFxBiIyYYGRsVKhI0LB0Qdy4fAUFTNikqKywhZDgv/EABsBAQACAwEBAAAAAAAAAAAAAAABAgMEBQYH/8QANxEAAgECBAMFBwMEAgMAAAAAAAECAxEEEiExBUFRE2FxgaEiMpGxwdHwBhThQlJi8RVyIyRD/9oADAMBAAIRAxEAPwC+0qTcre6NBwHJbaOY3qZ7pv0t9gguxum/S32CC7G6b9LfYILsbpv0t9gguxum/S32CC7G6b9LfYILsbpv0t9gguxum/S32CC7G6b9LfYILsbpv0t9gguxum/S32CC7G6b9LfYILsbpv0t9gguxum/S32CC7G6b9LfYILsbpv0t9gguyKcwGbC5PAcSVqy3OhBeyjXh2d0AgSLG2scVBZGzdjkPYISN2OQ9ggG7HIewQDdjkPYIBuxyHsEA3Y5D2CAbsch7BAN2OQ9ggG7HIewQEXkHIIC00fC3oPhbaOY9zNCAgCAIAgCAIAgCAIAgCAIAgCA8JQEPRfIHQc/1WqzpRehmoLBAEAQBAEAQBAEB6gIpAWej4W9B8LbRzHuZoQEAQBAEAQBAEAQBAEAQBAEAQGFZ0NceQPwjJSu7EYAtQ6YQBAEAQBAEAQBAeoAgIpAWej4W9B8LbRzHuZoQEAQBAceO2nSokio6IY6obGA1kSSdBrx4NcdAYhuxKi2af8AO6Vyc4GXNJY4CMhqf+LTbyS5OVmgdpsMfznR58JmKbWudA42e0wJN0zInIyTwuIbUY17fC4SOn98dCpKtWNyEBAEAQBAEAQGjGu7h84H3v8AaVWbtEyUVeaOBax0AgCAIAgCAIASgK9jO1FPdzSnObAEWb5k6H0WvKurabnfw/AqvbWrWyro9+7u7/QrdbH1X+Ko8+pH2C13OT3Z6OnhKFNWhBLyJnslXJqkOqv07rC4lp52PEW+6zUHrqzkccpKNFShBb6u2q6fEmFtHlCz0fC3oPhbaOY9zNCAgIvH48gljZBGp4+i0MTiZReSPxBwNrvFw53LXgtJYiotmLkjh9lsxP4r6Y3g7gqS9roAcO65sFo77tDrfgI2v+Qgl7ad+7/ZsUqU5q62Ol3ZymWtbkENywA+qPC0tAJmXDKSCDMzeU/5Kj0fp9zJ+2qdV+eRi3szSBBDS2xHdqVmiCGNIIa4B1mN1+lVfEqdtIslYafNo7MPs3dNDGDuiYuSTJLiSXGSSSSSTeVNPiVPKs6d+4ieFlfRnNtR1SnTc5rXSC38rnQ0uaHuDW+LK0udA1iFtxxVKa0kjD2E09UV7FbZxjQ8sw7nhoaWnduBflpOdVhmYua7PlDWOiYIBMyMt3yIyx5s347aGLYYZSdU/wBaS2me7DYoDLml2Z+paDYGYF1E6ih7zsIQUtjVh9qYwmmXUu45zW5t3VAM1nNkZoI/CAdmIAsTaWg0jXhKWVNXLSo2jexZis7dtzAtT1E7gIDlxzSQIBgGSeExAH3WCrUjfJfU2sPB+8caxm0EAQBAEAQBAcW18eyjTl4Lg45co1Mgz9pVKk1Fam7gMJUxNXLB2trcoFSJOWYkwDrHCfOFoM95HNlWbfn4imwuIa0EuJgAXJPkESbdkROcYRc5OyW7J7DbHxGGrUHuZIc6DlzOygw12cgQLOJ1iy2VRqU5JtHCrcUweNoVacZWstL2V2tVlu7vVdE9SaW1Znk8yLPR8Leg+FtI5r3OfG40MtEkj25SteviFS8SCMOOqRGY/affguc8XVatcXOPEOdlcWyXQY43i1iRN+EjqFhcnL3mL33OLY1au9jjiGBjpsAIkRqRLgDPAOPVJqKfslpqK90vzdoUGw0PaABaJiOui08knrY6SrUlomeHa9GYz+sOj3hT2ch+4p3tcVts4dgl9am0aS5wAnldRkl0NihF15ZaSzPey1M8JtOjVE06tN4GuVzTHW6hxa3RkqYerTdpxa8UdbTNxfooMT0MXMB1APoFKk1syGrnrWgaCFDberB49sggQDwJEgGImEBVG4/GZmOGGc0upuHh7rajn90HvaBtJxzZSPxm3Gh2pSTjlcrpd/519Ciik7pGDdpYq5qUX2ALstCtmBNOiRlIB3s1H1QcoIG7mwub06jpP2JeuhE4Rn7yNzMXX/DG6qZs9RrwaNYtOWs1tMTZoaaJe7eXbLRf8pvUxVSaactO6y5fHfkVjQhHVImOz2IqVKDXVm5XyQRkfTFjqGPAcB1A/U6VT3tGZlse43ZgN2WPLgf2W3QxjjpPVdSrj0Idwix1XTTTV0YzxSAgCAIAgKX2sqvNctce60AsHkQJPuD7LTrt5rHsuB06aw2eK1bebybt6M19ndiHFPIktY2MzuN9A3zt6JQouo+4vxbikcDTVleT2XLxfd8y+7J2LSw07sGTq50F0cgYsF0qdGNPY8LjuJYjGNdq9FyWi8fEkQspoEQqGYlKPhb0HwroxPc4dq4ae+Jka6RAn9Vp4ujmWZb/AEBEhcoqa8S4BjiTADSSb2EGTZFuStzj2NlIeWvc4TF21GgFtiAKkmQbG9iIsQVaZaRIKpQIDXiaLXtLXiW8f3HIojYwterQqqdF2l9+T7iikA8PeFkPrF2jbQrvZ4HvZee65zb87HVQ9dyk4Rn76T8VcsmA7d4pkB+SqP8AcMrv+TbfYrE6MXscutwXDz1jePqvg/uTlH+IlGO/QrA/7TTcPclvwqdg+pz5cBq39ma87r6M6sN2+wrj3hWp+bmgj/oXH7KHQlyMM+CYmPu2fg/ukWHA7QpVm5qVRjwNcpBjyI1B6rE4tbnMq0KlF2qRa8SGq7XxTamXcEtFbI4hj/8ASkzUa7NB7tXDm03ZWABju5FCNt+X59fQwnNhtt4ssBcwB2SbYfElrnfiZi0ZpaGBrDDoc/NlAaSFZ04X/lfn2FzXX2/jRIFARIDau6rFpaKtNhc5ky3Mx+dtz+afAZlU4dfUXZN7FxtWo6sKrYDSMpDKjAQS4avNzDQTAjvCCQQsU4pWsSQ3bDbdKhUa2M7yDmDSO7plzciQT7LcwlVxi09uR0MJwqpik5p5V1a38PA5aO1qLmB+8Y0H6i0EHkRK6CqRavc1qnDsTCo6eRt9ybXxN7MZTJyiowu5BzSfaVOZdTBLDVoxzSg0utnY8xGNp0yA+oxpPBzgD90ckt2TSwtaqs1ODa7lc3NcCARcHQjRWMLTTs9z1CD57tas51aoXkEhxbbQAGAB5Lnzbcnc+gYGnTp4eCprSyeu+vUuH8P2/gVDzqn7MYuhg/cfieP/AFQ//biv8F85FoW2ebCAiFQzEpR8Leg+FdGJ7nlekHNLTxVZxU45WQV+uIcRbjYcPJcjEp53+WJkaMTOR8NDzlMMMQ4we6ZtB09VrrchbnHsellDhu2ME6sp7oOteWSTbSZurTfeWkyQVShuwuHNRwaC0E8zAVW7K5eEHN2RIO7P1edM+rv2VO1RsftKiejRBVv4eVJOSrTA5EOMeUwrduuh7HD8dappVo3lza2ffqQHaLs/UwZYHuY4PByls/liQQeoWSE1I62Cx0MUm4pq29+8iFc3QgCA3YPFvpPD6T3McPzNN+h5jyNlDSejKVKUKscs1dFx2b/EJwAFekHc3sMH/gbT6hYJUFyZw6/Aot3pSt3P7/wXfZm0aeIpipSdmabciCNQRwKwSi4uzOBXoVKE8lRWZUf4i7VrUn0GUnvYILyWktkyABI1Avb/AHBZqMU02ztcFw1KpGcppPZa62/39CA/+a4zJk3jeWfI3ee+nrCydlC97HS/4jCZ82V+F9Pv6lfe4kkkkkmSSSSSdSSdSsp0kklZHiEiEFwhBvwuMfTc1zXO7pBiXQYMwQDp5eamMmndGGthqdaLjKK152V/HxLSztTS3Yc4OzzBYLxreTAI087raVeNrnmJcBrdq4xay/3P5WV3f0Kg5xJJOpuepWoetSSVkWXsRtYU3mi8gMeZbrO8OVoAgcQOPJbeEq5Xlez+Z5n9R8PdWmsRBXlHR/8AXV38n8y/LpHhwgIhUMxKUfC3oPhXRie5mhBXMQe87yJHtZcSvJyqO/INnNi3hrHkmAGuJMEwADJgXPQLCtwtzm2TlAcG1X1IN8xnLq3K0m8S06k9dFaXgTLwO5VKhAWHs9jXOmm4kkCWnjHEfCwVY21Rv4Wq37LJuFiNw+Z/xMqE4qm2bCk0xyLnvn4Hstqh7p6vgcUsPJ/5P0SKksx2QgCAIAgJHYu2auFfmpOsfEw+Bw8xz8xdVlFSWprYrCUsTHLNeD5r86F3r4rC7VohmYU67bsDvE13GPraeMX0sCFrpSpu/I4EaWI4bVz2zQe9tmvo/wA1RQNo4CpQqOpVW5XN9QQdCDxBWymmro9HRrQrQU4O6ZzKTKEAQBAEAQBAWLsNiIxBZAIe0m4kgskgg8NT9ltYSVp26nnf1LRUsKqt3eL66Wlv8kfQl0zwYQEQqGYlKPhb0HwroxPczQgr+OZFR48597/quLiY5arDOatOV0CTBgQDNtIJE+46rAgjk2W14Ds7GsuIhrWE2vIa949Z9OdpW5EytyO1VKmNRsgjmCOPEeV0JObYfZfEOE7xmUN3eU+Ejeiqb5cpPda27CI4G4M1KsV+fnzN7DLM3ImKvZSu5mV2IF20xoXQaVI02eKQZc5znGGm9iCAW4lWine342bdj57tttQ4yu5zw4Bz6YAzWDXnLc6wS+8XlZ4tZUkey4ZhpUqcZX0a28dV83y+RzqTphAEAQBAEAIQk9c4nUk9bqCEktjxSAgCAIAgCEkrsns9XxEFrcrD/wDY6wjm0au+PNZqeHnPXkcnHcZwuEvFvNJf0r6vZfPuL5sTYtPDNhvecfE8gZj5eQ8l0qVGNNaHh+I8SrY2eaeiW0Vsvu+/6EmspzggIhUMxKUfC3oPhXRie5mhBD7XZDweY+4/sLl46NpKQI6tTzNc0zDgRYwbiLHgVpJ2C0OfAYBtHNlLjmMnNl1i5sBrr9tAAJlK5Llc6lBU24cNzNzmGzfXT0UO9tC0LZlm2LFQ2xQAyiWhundMHpH6rA6cjoRxNJaLQjtqdsadEA7t7iSYEtGnPl91KotnR4bh3j5SUXbLbfvPmGIq53vedXOc49XEk/K2ke6hHLFR6JL4GnOJiROsSJjnCmxOaN8t9egc8AgEgE6CRJ6c0sHKKaTe5654GpAm17X5IHJR3YJQltLcSgutxmGvDmgurXDXA6EFCE09j1CQgCAIAhJI4LYeIqwWUnQfzGGjrLon0WWFCpLZHOxHFsHQbU6iv0Wr9PqWTYXZF1Oo2pXLHBtwwSRm4SSBprHOFt0cI4yvI85xL9RRrUnSw6avvJ6ad1r7/It63TygQBAEBEKhmJSj4W9B8K6MT3M0IIvbT/AOp+P79FoY96JDkRGJbLHDm0jQngeAuegXNW4W5Hdn8MKbXNDy8yCZY+mRa3dfeORV5u5abuSqoUNeIrtY0ucYA4oZsPh6mIqKnSV5MhsT2iExTZPm4x/1HD1CtlPUYb9LNq9edu6Kv6v7MhsXi31DLzPLgB0Ck9RhMFQwsMlGNlz6vxf53GhSbJodhgagqSZAAjh+bhz72v7lTm0sYXQTqqrfVfDn667+XNmVSjL2umMsiL3mOR8uMonpYmdLNOM77X9bd/d3nmJoZ4uR5jkYn4ROwq0u0trY9xVAPblNrzI1BGhHmDf0ROzFakqscrPXUpblngL9Ij4UX1uS6d4ZfzQ8FHuZJ4ET14qb63HZ/wDjyd1jHCYYUwQCTeb9AP0SUrlaFFUo5Ub1BmCAIDJjZIEgSYkmAPMngESu7FZzUIuTvprpq/JH0vYuwaNFrDlY+oL70gEyRfKeA5BdelQjBLr1Pm/EOK4jFTleTUX/AE30t3rm+t+ZLrMcsIAgCAIAgIhUMxKUfC3oPhXRie5mhBB7UdNQ+QA/X9VycbO9S3QN8jireF1psbazbSOK1EEcOxAcri6i2iSZygQTbV3CehI81afjctPxuSCqUKltzFF9VzZ7rTAHCRYnrMq6Po3A8HToYWE0vakrt89dUvC1vmR6k7IQBAEAQBAEAQBAEAQBAEBaeynZ1tZm9qzlzQ0XuGyHEwedvQ+S3MNh1JZpHluOcYnQqdhReqWr03e2/dr5rvReKVJrWhrQGtAgACAByAXRSS0R4yc5Tk5Sd2+ZmhUIAgCAIAgIhUMxKUfC3oPhXRie5mhBXK5lzjrc39VwqzvUl4hnNi43b5cWDK6XjVtj3h5jX0WNbhbnNsunlzzUzkmSBnhuoMZ3OOoPGLC3E2kTI347Fikwud0A5ngFVI2sBgp4ysqUPFvoub/OZSZ53PNZD6nZLRBAEAQBAEAQBAEAQBAEAQEn2f2ScTVy3DG3e4ESAZgDzJELNQpOpK3LmczivEY4Kjm/qfurv0v5K/0Pp1Gk1jQ1oAa0AADQAaBdZJJWR82nOU5OcndvVszUlQgCAIAgMK1UNaXO0AJOpsOiFoxcpKK3Zx4Pa9Kq/IwmYkSCJ6ShsVsHWpRzTWhzqhjJSj4W9B8K6MT3NeMq5WOPGLdTZY608kHIgr64RBqxfgfDQ/uu7hiHWPdM89PVFuStzh2Wd3TqOdRZRAJJDWgSGz37cxeOExJV5ave5mhSlWqRpwd22kvFkBtLHGs+bhv5W8vM8JRKx9H4Zw6GCoqCs5c31/hcvicik6JqpB2Z2bw2y6cbmemil2sYoZ88s23L6/Y01d5u2ROe2bw8ryTYXi4nopVrsxT7bso297S+3TX16X8LHTfN5R95H6KvI2Nc/dY04bP3s0+UlsTfQDQaa3Uu3IxUe19rN5bfTl46nmGD8pzZgeE5c3hGuW3izekKXa+hFJVMjz3vy2vsumm9/IVhUyNynvcSY+k/+0IrX1FRVezSh73P4P62PcTn7uWfOMustiZ/LGaYvooVuZNbtfZyee2+m9+W97a7WFUPztjwcdPP+iK1hNVO1i4+7z9f4PQX5zY5dZ7saNEc5mSmliV2na/4+Xdp16mFAVM7s3g70eHmI08pUvLbQpS7btXn93W23XT0OpVNkID6d2a2SMPRAI/EdDqh8+DejZj3PFdehS7OPfzPmvFuIPGYhyT9laR8Ovnv6ciWWY5YQBAeICB2xt8N7lEgu4vsQOnM/ZSkdXB8Oc/bq6Lpzf2IrZ21jTe578zzBiSLH14dEsb+JwMakVGFo/n5uWzDVN7TaXNjMLtN7HgeYUHn6kezqNRez3NL9lUi9lQDK5pB7sAGOBCGVYuqoOm3dPrr8DmVChKUfC3oPhXRie5B7b2iGkh7mta3hNyefn0XLxdWUpZOhnw+ErYiWWlFv5eb2XmVnFdouFNvq79h+61Mp6jCfpezviJ+Ufu/ovMhnYt5fnzuzc5+3Ty0UnpY4LDxo9ioLJ0t6+Pfubq21Kr25XPsdbNE9YCWNWhwfBUaiqQhqttW7fFnGpOmEAQBAEAQBAEAQBAEAQBATOxKbGupPe0EZw4zwAIv+q6eGox7NSa13PP8Vr1JKdOm+TWnPTX7H0tbR4EIAgCAICMxWxqbmVGsa1rnQZjQiNOQtw5lDcpY2pGcZSbaXK5EYTs88VQH5SxpBJ4O/wBoH2Km50avE4Ok8l8z08O/7FpCg4R6gIhUMxKUfC3oPhXRhZ8oxZdWr1C2Xl73ERckSSIjhlXFlec3bqfU6GXDYWCl7KjFX5a2X19TllUNpJ8whIQBAEAQBAEAQBAEAQBAEAQHXgcPmMkW+StvC0M7zS2+ZoY7FdmskHr8iSXVOGXXY+1GVWNBcBUiC06kjiOY4qh5jF4SdGb09nkySQ0zTiqpaxzmtLiNGjUoXpxUpJN27yP2I6u7M+s6xsGQBpxjhyuht4xYeNoUltu7kshohAYtaBp5n1Jk/coS3cyQgICIVDMSbGywA6Fse4VuRiu07opG3dmtwT6bsPOZ4e0Alxe090ZmR5Ej16LQq01RacOZ7Ph+MnxOnOGKfsxcW9Eota6O/er+RW8Th3sMPaQfPXzWnKLi9T0lDEUq0b0mmu41KDMEAQBAEAQBAEAQBAdey9mVcQ/JRaXHUnRrRzc7gP7CrKSirsw18RToRz1HZfPwLQ3+HlbLJrUg7lDyJ/m/osXbrocl8epX0g7eXy/krW19k1cM/JWbBIlpBlrhxLT+mvuFljJSV0dXDYqliI5qb+68TTgWgvAIlbOGjGVRKSKY2Uo0W4ux1v2gwGoCTNMS7TSAf1Gq690jzjqK7vyM6eLa4sDcxzxECfE4NbPVxAH7JmRWVWMVmb0J/YODYXB9QuBa8MDR9eZzbuB4FpkdOajMcnG4/NFwp7Nav+PqTtHbVJ7C9mZ3ey5WtOYuvLQOYyuPQKLnIys3YradOm17nkgMLQ6xsX5YtrADgSeABJ0S5Ci2bqFAtLu8TJJgxYk8I4RA9FJaUlK2ljchQIAgCAICIVDMSlHwt6D4V0YnuZIQfOe20f4t0R4WZv5o4/8A5yrl4u3aH0L9OZv2KzdZW8P93IJax3AgCAIAgCA34HBVKzxTpML3m+URoNSSbAeZUNpK7MdWrClHPUdkSuM7JYukwvdSkC5DXNc4DnA19JVFVi3Y1KfFMLUnkUte9WIIm0hZDoJan0nEdlMBRDBVbUzEXIdWPhgPe4Ns1oJEmwGYLWVSb2PIS41iruzVvBEhsnbGCZTDaEtZIFqdWASGZS8xq7eUwCbkujgYrKnNvU5tWvOtLPUd2bXdq8KGhxe6CGuByVIIfU3TYMcXH2vpdR2MzHc8xuOweIbu6wzAvawBzKg77nPpgsdH1MeMwNoMwEUZx1Rlo4ipRlmpuz/PiVTb3ZZuFcKlN5LHEtDXDvNJBPiGogHhPVdLh0s9TXkjtLiksRS7OUdevJ+REGi3vW8Rk+ZgD9AuzYwWRY+zOw6Ya2q5gtBp3dYAmxbMESAQDMQCIgKrSODxKunLs48tyfZgqYLiGAFz94Ym9SAMx84A9kscy7NP+UUcrmhkBxaXZS9pJa0NBzNIIOUASDpZRZE5mZYrZlGoC2pTa9pMlrhLSQ0sEt08Jj76pZEKTR1gKSD1AEAQBAEBEKhmJSj4W9B8K6MT3M0IPmna+iG4upE3yuMkm5AnXguTilaoz6LwCo54GF+V15X/AD/ZDLAdkIAgCAIAgLJ/DralFmIfnqUwHUnkOLmizCHOv/KHO6MPJYq0JNbHC41OM8PFxknr9GfSKe1qDiGtrUi60AObN3Bgt/M5repAWtkl0PMHybtYKLMVV3T6ZpuyvbBBH4gFh1dMdQtynmyq57PhuKVTDxcparR/Gy+OnmWLsrtqnWyUMY5xc0xSqFz25gYmlUc0jMCQLOkOgA3AnFOLWsTm8V4Z/wDaitOaXzX1Xn4W9nZ3DtygMIDYhuerl7oYGy3NBjdsN/zDNqSTh7SR56wb2ew4Y5gYYcWkuz1c5Lam+ad5mzf6ne1unaSvf86Cxy7Q2TgqYLqzRqXDPUqucHHOc1MFxLXS95BbBBcSslJVqjtD5fMtGm5bIqnafblTEg5RlDQ4sFi7NFiZIBPlYefFdzC4RUE3e7f5odCnR7NO25CU21XUobO9LSBJYDmvBmIHA6enBbeti05uFJyfJF9p0qtPChrIdVDAGkZYzGwcZABgmTYTBsNFXWx5WcnOblIyqPrnDAtAbXLWTIBaHyA8xxbqeia2K6X7j2nUrHDscWltU5C5vcJbL2522sYbmuOVk1sNLjalSsH0dy0lueavh8FmFt7z389r/hRxudwra3PNourb6huy8U5dvMoYeNPKHZmkgQX6RpqjuFazJFSVCAIAgCAiFQzEpR8Leg+FdGJ7maEHzPtdVa7F1cvDK0nmWtAP7ei5OJadV2Po/AacoYCGbndrwb0+/mQ6wHXNlKg5xsPXgslOlKbskYqteFJXk/Lmdg2cOLvstxYFc5ehznxN20j6/wAHj9ncne6SwP8AayYcT/uj8DkrUS3UevBadSlKm7SR0KNeFVXizU4SI5rGZWrqzNmxsPTbWYaz6m7ILHnuzlc17AbN/LnJ9OKSk7aHNxmCdWhKMZNuy6a2d1y8drd59Wo9naUBzK1XJIqMINEta4Oa9rw7Ic0ZQO8TIkmScy1O1fNHjnBxeVrXofNe1+BwxrBtCpUc1jQ0ulkS0EDK5rQXROpm/ktqnOVrs9Pw3h0o0W6t1d3S2tpu/scBQ7pa9n7exLKTGCq6APzBrnXvEuBNl0aWBoyinKOvmeWxdKlKtJqK35bG1+38SRBrO9MoPuAsywVBf0o11Rp9CNc4kkkkk6k3J6lbKVlZGUm9jbCL4fVswiQJgnlPIfdQ2crGcRULwpb830JluwqA/JN5uXe2twoOa8fiH/V8iRa0AAAQBYDyQ1G76s9QgIAgCAIAgCAIAgIhUMxKUfC3oPhXRie5k42MCTyQLfUqFLsa6oXPr1u++XQ0SA4mbuce8PbqtFYRy1m9Weql+pI0lGnh6fsxsrt7pdy2b8/A01+xLmiWVQ+L5S3LI5TJvKLBJNNvQyx/VCl7Mqdu9O9vKyJHZfZ381bTgwH/AMiNOgW/focnFcUd7Uvj9v5JX/JaGXLux1vm/wCUyoOf+9r3vmf0+BB7W2CaYL6ZLmi5B8QHPzClM6uE4iqjUKmj68n9iDc0EQbhJRUlZnWhNxd4vUjMThC02kj46rlV8NKm7rVHdw2MjVVpaP8ANjUyi4mACsMac5OyRsTrU4K8md1LBd3K5zoNy0Ehs9OK3oYGO8nqcqpxGTleCXi9zfTw7W6AfJWzChTgtEatTE1aju38NDYBCyKKWyMLk3uwrFQgOvZeG3lVjYtMn+UXM/HqoZr4ur2VGUvh4v8ALl8VTygQBAEAQBAEAQBAEB45wFyQB5oDAV2/U33Ci6JyvoRG+b9TfcKtzNZkxR8Leg+FdGF7maEBAEAQBAYVWktIESQQJuNOIQmLs02UDEUHU3FrokciCPcKyPXUqsakc0djUpMgQHdsrZzq7oFmjxO5f1UNmri8VGhG/PkibZ2Yp8XvPTKP0Ki5y3xaryivU8f2XZaKjxzkNNvLSPulwuLVOcV6mb+zNKLOqA8zlI9oCXKrita+qX55m/Z+wqdNsPDajpmSOHIC6GLEcQqVJXi3FdEzo2dsxlHNkmXHU8BwaPJQYsRiqle2bl+XO1DWCAIAgCA8c4DUgIDV/imfUPS59gozLqX7OXQxdjG8JPof1hVdSJZUZvkYPxgiwM+fD9/RQ6itoWWHlfU5t476nfrP98NFjzyNjsYdDxzidST/AHyFlVyb3LRhGOyCguRSAs9Hwt6D4W2jmPczQgIAgCAIAgK72j2U5zhUptJJs4ASfJ0fb2UpnX4djIwTpzdlun9CuNaSQACSbAcZ5KTtuSSu9i4bE2YxlNji0F5E5jcibgDlbkqnm8bip1ZuKfs9CUAQ0T1AEAQBAEAQBAEB4SgOSrjPpHqdPQLHKp0NiGHb945XXMm55rC23ubUYqKsgoLBAEAQBAeoCKQFno+FvQfC20cx7maEBAEAQBAEBi9wAJOgQlK+hF4inSzbxtIOeLgnugnWT5yNYVHURu03Xcezcmo/H88CTpOloJESAY1+6uaTVnoZoQEAQHiA9QBAEBi94FyQAlyUrmitig0gATzvoP1tdUlNIyQpOSbNOKrh0ATGpsR0F/f2VJzTVkZqNJp3kaFiNkIAgCAIAgCA9QEUgLPR8Leg+Fto5j3M0ICAIAgCAIDmxzu7HM/YX/YeqpUdkZqEbzOJa5vHVgq0Qw+n7LPTlyNOvTs8yOxZDXCAIDFzwCBxMx6aoTYyQgIDwmLnRARbnFxzHX48lrSldnRhBRVjwBVLhAEAQBAEAQBAEB6gIpAWej4W9B8LbRzHuZoQEAQBAEAQEfjHS/oI9Tc/p7LBUetjcw8bRuaVjNgFAbWYlw4g9dfdZFUaMEsPF7HjsQ8/mjoLfeUdRkqhBHjKzho4+t/79FCqNEuhBmD3OJmTmFwT8Ry6f1UZne7JdKOXKjp/xpgd2/G9vQ/0WTtUYP2z6mRxo+l3/WPeVPaor+3kc1R5cZPtwHT91ilJs2YU1BGKqZAgCAIAgCAIAgCAID1ARSAs9Hwt6D4W2jmPczQgIAgCAIDF7oBJ0F0JSu7EZfU6m59brVbu7nSisqsFBIQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQEUgJKnoOgWwaR6hAQBAEAQGrFeE+nyFEti8PeRwrAbYQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAcSEH//Z

Reply:
Ok, although the borders are not precise there.

Reply:
Could you look at cities here too in García Cortazar's book too, maybe?

Reply:
I don´t have it, although I could search in my nearby libraries.

Reply:
I have the book but just shows Leovigild's conquest in 577. All in all, observing the lands marked as "conquests" the zone of Aracena doesn't enter in it.

Reply:
Okay, thank you.